docs/Node Operators/configuration-variables.md

```diff
@@ -481,7 +481,7 @@ It is not recommended to change this unless you know what you are doing.
 
 - Default: `"6688"`
 
-Port used for the [Chainlink Node API](../chainlink-node-api-reference/) and GUI.
+Port used for the Chainlink Node API, CLI, and GUI.
 
 ### SECURE_COOKIES
 
```

Comment:
Maybe just specify in the [CLI section](https://docs.chain.link/docs/configuration-variables/#cli-client) that one can run `chainlink help` to get more details. The same explanation is provided on [github](https://github.com/smartcontractkit/chainlink#running-a-local-chainlink-node)

Reply:
Good suggestion. Done